views/order_view.py

```
@@ -0,0 +1,118 @@
+    """
+    try:
+        # 查询订单
+        order = order_service.get_order_by_id(order_id)
+        
+        if order is None:
+            return json({"error": "订单不存在"}, status=404)
+        
+        # 返回成功响应
+        return json(order)
+        
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)